scripts/host-check

```diff
@@ -1749,16 +1793,37 @@ def run_checks_all_platforms(extra_checks: List[str]) -> bool:
         )
     if platforms_errored:
         print(
-            "!! One or more platform checks could not be performed, see errors above !!"
+            "!! One or more platform checks could not be performed for XR platforms:   !!"
+            + "\n      "
+            + ",\n      ".join(sorted(platforms_errored))
+            + "\n   See errors above."
         )
+    if platforms_warned:
+        print(
+            "!! One or more platform checks resulted in a warning for XR platforms:    !!"
+            + "\n      "
+            + ",\n      ".join(sorted(platforms_warned))
+            + "\n   See warnings above."
+        )
+
     # Print the result of any extra checks
     if extra_checks:
-        print_extra_checks_summary(
-            extras_supported, extras_not_supported, extras_errored
-        )
+        print_extra_checks_summary(extra_results)
     print(f"{DOUBLE_DASHED_LINE}")
 
-    return len(platforms_supported) > 0
+    if (
+        len(platforms_not_supported)
```

Comment:
Presumably this can just be `if platforms_not_supported or ...` like above, rather than checking length?